src/components/Modals/FoundAllocationModals/FoundAllocationModals.tsx

```diff
@@ -0,0 +1,68 @@
+import DarkOverlay from "components/DarkOverlay/DarkOverlay";
+import styles from "./FoundAllocationModals.module.scss";
+import ChooseFoundAllocationModal from "./ChooseFoundAllocationModal/ChooseFoundAllocationModal";
+import FoundAllocationModal, {
+  putAllocationId,
+} from "./FoundAllocationModal/FoundAllocationModal";
+import { DragDropContext, OnDragEndResponder } from "react-beautiful-dnd";
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { AllocationTypes, updateAllocationAction } from "store/allocationSlice";
+
+interface FoundAllocationModalsProps {
+  allocationId: string;
+}
+
+const FoundAllocationModals = ({
+  allocationId,
+}: FoundAllocationModalsProps) => {
+  const [percentageInputValue, setPercentageInputValue] = useState("");
+  const [amountInputValue, setAmountInputValue] = useState("");
+  const [sizeInputValue, setSizeInputValue] = useState("");
+  const dispatch = useDispatch();
+
+  const handleOnDragEnd: OnDragEndResponder = (result) => {
+    const type = result.draggableId as AllocationTypes;
+    let isValueValid = true;
+
+    const switchValue = {
+      [AllocationTypes.AMOUNT]: amountInputValue,
+      [AllocationTypes.PERCENT]: percentageInputValue,
+      [AllocationTypes.SIZE]: sizeInputValue,
+      [AllocationTypes.CAPITAL]: null,
+    };
+
+    if (type !== AllocationTypes.CAPITAL && !parseFloat(switchValue[type]))
+      isValueValid = false;
+    if (result.destination?.droppableId === putAllocationId && isValueValid) {
+      dispatch(
+        updateAllocationAction({
+          id: allocationId,
+          type,
+          value: switchValue[type],
+        })
+      );
+    }
+  };
+  return (
+    <>
```

Comment:
unnecessary react fragment, there is already one react child

Reply:
There is two react children: div with class wrapper and darkOverlay component so react fragment is necessary here